packages/commonwealth/client/styles/pages/login/login_mobile.scss

```diff
@@ -61,15 +61,15 @@
 
 .LoginMobile {
   @include mediumSmallInclusive {
-    background-image: url('/static/img/login-background-mediumsmall.png');
+    background-image: url('../../../../static/img/login-background-mediumsmall.png');
```

Comment:
Given this is on the client, not the server, what's the undesired side effect of leaving these imports as relative?